src/frontend/pages/game.tsx

```diff
@@ -91,30 +103,50 @@ export default function Home(): ReactElement {
         <div className={styles.actionBar}>
           <div className={styles.actionBarLeft}>
             {/* TODO: both of these values will be given by an API/socket */}
-            <h4>You Are: </h4>
-            <h4>Current Coins: </h4>
+            {/* TODO: add some JS to change styling of the color based on what color it is  */}
+            <h4 className={styles.actionBarText}>
+              You Are: <span style={{ color: 'green' }}>{playerColor} </span>
```

Comment:
Can we use something like `playerColor.toLowerCase()` instead of `color: 'green'` that way it's more responsive?

Reply:
yep this is smart, I was going to add a separate helper function but this is much cleaner, thanks :)